pennylane/transforms/optimization/cancel_inverses.py

```diff
@@ -240,6 +242,11 @@ def eval(self, jaxpr: "jax.core.Jaxpr", consts: list, *args) -> list:
             self._env = {}
             return outvals
 
+    @CancelInversesInterpreter.register_primitive(measure_prim)
+    def _(_, *invals, **params):
+        _, params = measure_prim.get_bind_params(params)
+        return measure_prim.bind(*invals, **params)
```

Comment:
Just to be consistent

```suggestion
        subfuns, params = measure_prim.get_bind_params(params)
        return measure_prim.bind(*subfuns, *invals, **params)
```